code/datums/status_effects/debuffs.dm

```diff
@@ -136,3 +136,437 @@
+	. = ..()
+	if(. && (needs_update_stat || issilicon(owner)))
+		owner.update_stat()
+	owner.update_canmove()
+
+
+/datum/status_effect/incapacitating/on_remove()
+	if(needs_update_stat || issilicon(owner)) //silicons need stat updates in addition to normal canmove updates
+		owner.update_stat()
+	owner.update_canmove()
+	return ..()
+
+//STUN - prevents movement and actions, victim stays standing
+/datum/status_effect/incapacitating/stun
+	id = "stun"
+
+//IMMOBILIZED - prevents movement, victim can still stand and act
+/datum/status_effect/incapacitating/immobilized
+	id = "immobilized"
+
+//WEAKENED
+/datum/status_effect/incapacitating/weakened
+	id = "weakened"
+
+/datum/status_effect/incapacitating/paralyzed
+	id = "paralyzed"
+	needs_update_stat = TRUE
+
+/datum/status_effect/incapacitating/sleeping
+	id = "sleeping"
+	tick_interval = 2 SECONDS
+	needs_update_stat = TRUE
+
+/datum/status_effect/incapacitating/sleeping/tick()
+	if(!iscarbon(owner))
+		return
+
+	var/mob/living/carbon/dreamer = owner
+
+	if(dreamer.mind?.has_antag_datum(/datum/antagonist/vampire))
+		if(istype(dreamer.loc, /obj/structure/closet/coffin))
+			dreamer.adjustBruteLoss(-1, FALSE)
+			dreamer.adjustFireLoss(-1, FALSE)
+			dreamer.adjustToxLoss(-1)
+	dreamer.handle_dreams()
+	dreamer.adjustStaminaLoss(-10)
+	var/comfort = 1
+	if(istype(dreamer.buckled, /obj/structure/bed))
+		var/obj/structure/bed/bed = dreamer.buckled
+		comfort += bed.comfort
+	for(var/obj/item/bedsheet/bedsheet in range(dreamer.loc,0))
+		if(bedsheet.loc != dreamer.loc) //bedsheets in your backpack/neck don't give you comfort
+			continue
+		comfort += bedsheet.comfort
+		break //Only count the first bedsheet
+	if(dreamer.get_drunkenness() > 0)
+		comfort += 1 //Aren't naps SO much better when drunk?
+		dreamer.AdjustDrunk(-0.2 * comfort) //reduce drunkenness while sleeping.
+	if(comfort > 1 && prob(3))//You don't heal if you're just sleeping on the floor without a blanket.
+		dreamer.adjustBruteLoss(-1 * comfort, FALSE)
+		dreamer.adjustFireLoss(-1 * comfort)
+	if(prob(10) && dreamer.health && dreamer.health_hud_override != HEALTH_HUD_OVERRIDE_CRIT)
+		dreamer.emote("snore")
+
+
+//SLOWED - slows down the victim for a duration and a given slowdown value.
+/datum/status_effect/incapacitating/slowed
+	id = "slowed"
+	var/slowdown_value = 10 // defaults to this value if none is specified
+
+/datum/status_effect/incapacitating/slowed/on_creation(mob/living/new_owner, set_duration, _slowdown_value)
+	. = ..()
+	if(isnum(_slowdown_value))
+		slowdown_value = _slowdown_value
+
+/datum/status_effect/transient/silence
+	id = "silenced"
+
+/datum/status_effect/transient/silence/on_apply()
+	. = ..()
+	ADD_TRAIT(owner, TRAIT_MUTE, id)
+
+/datum/status_effect/transient/silence/on_remove()
+	. = ..()
+	REMOVE_TRAIT(owner, TRAIT_MUTE, id)
+
+/datum/status_effect/transient/jittery
+	id = "jittering"
+
+/datum/status_effect/transient/jittery/on_apply()
+	. = ..()
+	owner.do_jitter_animation(strength / 20, 1)
```

Comment:
this isn't the case, this would be the same as strength / 10

this used to make you jitter for the number of cycles you had of jitter, now it goes off a value that is 20* more than that, so it needs to be divided by 20.